fdbclient/admin_client.go

```diff
@@ -529,6 +529,11 @@ func (client *cliAdminClient) ChangeCoordinators(addresses []fdbv1beta2.ProcessA
 
 // GetConnectionString fetches the latest connection string.
 func (client *cliAdminClient) GetConnectionString() (string, error) {
+	if client.Cluster.UseManagementAPI() {
+		// This will call directly the database and fetch the connection string
+		// from the system key space.
+		return getConnectionStringFromDB(client.Cluster)
+	}
```

Comment:
Do you think it make sense to return here only the connection string without using `ParseConnectionString` and make use of the `ParseConnectionString` only once in this method?